src/test/java/com/github/cameltooling/lsp/internal/completion/CamelKubernetesServicesCompletionTest.java

```diff
@@ -39,13 +44,19 @@
 @EnableKubernetesMockClient(crud = true)
 class CamelKubernetesServicesCompletionTest extends AbstractCamelLanguageServerTest {
 
+	@Inject
 	private KubernetesClient client;
-	
+
+	@AfterEach
+	void cleanCluster() {
+		KubernetesConfigManager.getInstance().setClient(client);
+		client.namespaces().delete();
+		client.secrets().delete();
```

Comment:
i think the injected mocked client should be removed in the @AfterEach (or @After depending were it is created)